### PR TITLE
Support pre-release versions for ent accept tests

### PR DESCRIPTION
**Pull request by lkysow**

### CODE REVIEW

File: test/acceptance/framework/config/config.go
Comment (ishustava) [inline test/acceptance/framework/config/config.go:91]: This cast is probably safe here since we know it's always a string, but I'd still check and error just in case.

File: test/acceptance/framework/config/config.go
Comment (ishustava) [inline test/acceptance/framework/config/config.go:100]: I think we should add a test for this.

**Review by ishustava — APPROVED** [review]
Looks great! A couple of comments, but I won't hold up the merge on that!

File: test/acceptance/framework/config/config.go
Comment (ashwin-venkatesh) [inline test/acceptance/framework/config/config.go:93]: Do all our pre-releases share the semver `x.y.z-rc*` ?

**Review by thisisnotashwin — APPROVED** [review]
This looks good and can be merged after addressing Iryna's comments

File: test/acceptance/framework/config/config.go
Comment (lkysow) [reply test/acceptance/framework/config/config.go:93]: Yeah afaik. We have `-rc<n>` and `-beta<n>`

File: test/acceptance/framework/config/config.go
Comment (lkysow) [reply test/acceptance/framework/config/config.go:104]: @ashwin-venkatesh, test added. 👍 ?

Comment (ashwin-venkatesh) [reply test/acceptance/framework/config/config.go:104]: :shipit: looks good!!

**Pull request merged**
